index.html

```diff
@@ -1375,10 +1362,10 @@
             Reconnecting to a presentation
           </h4>
           <p>
-            When the <code><dfn for="PresentationRequest" data-lt=
-            "reconnect()">reconnect</dfn>()</code> method is called, the
-            <a>user agent</a> MUST run the following steps to <dfn>reconnect to
-            a presentation</dfn>:
+            When the <code><dfn for=
+            "PresentationRequest">reconnect</dfn>()</code> method is called,
```

Comment:
OK, I hadn't done it because I did not want to look too picky, but I agree :) I removed all parentheses. Even when "method" does not appear afterwards, the reference is styled as code and links back to the method definition in any case.